integr8ly/integreatly-operator
# Fix Service monitors

# Issue link

https://issues.redhat.com/browse/MGDAPI-3026

# What

Add messing service monitors and set the name space prefix


# Verification steps



* install the operator 
* Run `BYPASS_STORAGE_TYPE_CHECK=true INSTALLATION_TYPE=managed-api TEST="Verify servicemonitors are cloned in monitoring namespace and rolebindings are created" make test/e2e/single`
* Expected: Test should pass
